builtin/game/chatcommands.lua

```diff
@@ -1056,3 +1056,44 @@ core.register_chatcommand("kill", {
 		return handle_kill_command(name, param == "" and name or param)
 	end,
 })
+
+local spawn_spawnpos = minetest.setting_get_pos("static_spawnpoint")
+
+minetest.register_chatcommand("spawn", {
+	params = "",
+	description = "Teleport to the spawn point",
+	func = function(name, param)
+		local player = minetest.get_player_by_name(name)
+		if not player then
+			return false, "Player not found"
+		end
+		if spawn_spawnpos then
+			player:set_pos(spawn_spawnpos)
+			return true, "Teleporting to spawn..."
+		else
+			return false, "The spawn point is not set!"
+		end
+	end,
+})
+
+minetest.register_chatcommand("setspawn", {
```

Comment:
I don't see much potential in this command. Server owners can just use `/set` to update the spawn position. As a result, you may not use the variable `spawn_spawnpos` to avoid the cache being outdated.

Reply:
Well, I can not use 'spawn_spawnpos' variable. I think this command is simple and intuitive. /home, /sethome. /give, /giveme. /spawn, /setspawn. No one likes big command with several variables, it is difficult!